up42/catalog.py

```diff
@@ -358,54 +359,58 @@ def search(
         else:
             return df.__geo_interface__
 
-    @staticmethod
     def construct_order_parameters(
+        self,
         data_product_id: str,
         image_id: str,
-        aoi: Optional[
-            Union[
-                dict,
-                Feature,
-                FeatureCollection,
-                list,
-                GeoDataFrame,
-                Polygon,
-            ]
+        aoi: Union[
+            dict,
+            Feature,
+            FeatureCollection,
+            list,
+            GeoDataFrame,
+            Polygon,
         ] = None,
     ):
         """
-        Helps constructing the parameters dictionary required for the catalog order.
+        Helps constructing the parameters dictionary required for the catalog order. Some datasets have
+        additional parameters that are added to the output dictionary with value None. The potential values to
+        select from are given in the logs, for more detail on the parameter use `catalog.get_data_product_schema()`.
```

Comment:
catalog.get_product_schema is an endpoint, should we reference the endpoint better?. this method does not have too much information.

Reply:
How do you mean? Explain what it does here? Hm not sure, it should just be a small note that this function might be relevant, has its own docstring.